documentation/modules/ROOT/pages/connectors/cassandra.adoc

```diff
@@ -773,6 +773,10 @@ As the connector generate change event, it will publish those events to Kafka us
 
 Depending on the write load of a table, when a Cassandra connector is stopped for a long time, it risks into hitting the cdc_total_space_in_mb capacity. Once this upper limit is reached, Cassandra will stop accepting writes for this table; which means it is important to monitor this space while running the Cassandra connector. In the worst case scenario when this happens, the best thing to do is to (1) turn off Cassandra connector (2) disable cdc for the table so it stops generating additional writes (although writes to other cdc-enabled tables on the same node could still affect the commitlog file generation given the commit logs are not filtered) (3) remove the recorded offset from the offset file (4) once the capacity is increased or the directory used space is under control, restart the connector so it will rebootstrap the table.
 
+==== Cassandra Table CDC is Enabled, Then Temporarily Disabled, And Then Enabled Again
+
+If a Cassandra table temporarily disabled CDC and then re-enabled it again after some time, it must be re-bootstrapped. To re-bootstrap an individual table, you can remove the recorded offset line from offset.properties corresponding to that table.
```

Comment:
Re offset.properties, so that's not something stored in Kafka (yet)?

Reply:
> Re offset.properties, so that's not something stored in Kafka (yet)?

I don't think so, since what we store in kafka is about a single change event from cassandra, but this offset property is about the whole cassandra-connector. `snaphsot_offset.properties` is a property file created by cassandra-connector upon started, and it stores which tables have been snapshotted by cassandra-connector already. 

The `snaphsot_offset.properties` file would look like below:
```
testcassandra01.bar=\:-1
test_cdc.table_c_test=\:-1
test_cdc.test_kafka=\:-1
testcassandra01.foo=\:-1
test_cdc.table_b_test=\:-1
test_cdc.table_a=\:-1
testcassandra01.test=\:-1
```

Reply:
Ok, thanks for clarifying. We might consider storing this in a Kafka topic, akin to the offset topic of Kafka Connect (used by the other connectors in distributed mode). One bit of local state less on the nodes. But something for another time, if you like.

Reply:
> Ok, thanks for clarifying. We might consider storing this in a Kafka topic, akin to the offset topic of Kafka Connect (used by the other connectors in distributed mode). One bit of local state less on the nodes. But something for another time, if you like.

@gunnarmorling I see, a minor concern about storing this in kafka instead of local files would be that we don't seem to be able to modify the offset manually thus cannot trigger re-snapshot on tables arbitrarily.